apps/web/contexts/Analytics.tsx

```diff
@@ -12,15 +13,15 @@ export type AnalyticsContextProps = {
   fullContext: string;
 };
 
-export const AnalyticsContext = createContext<AnalyticsContextProps>({
```

Comment:
let's change this name back to AnalyticsContext. This is the convention that makes the most sense here